WordPress/gutenberg
# Update `RNTAztecView.podspec` dependency

## What?

Updates `RNTAztecView` to use the latest Aztec iOS build https://github.com/wordpress-mobile/AztecEditor-iOS/pull/1376

## Why?
To address a bug in iOS 17.

## How?
By using `~> 1.19`. While this does _explicitly_ use 1.19.9, it will let every client update to it as long as its upstream dependencies are compatible. 

## Testing Instructions
I'm not sure about this. The Xcode project in the module doesn't build in isolation because it depends on React files that are only available when it's imported. 🤔 

~~Maybe I should create a PR in gutenberg-mobile that uses this Gutenberg version and ensure it works.~~ Update: I created https://github.com/wordpress-mobile/gutenberg-mobile/pull/6201 to test this change works in gutenberg-mobile, when build on top of https://github.com/wordpress-mobile/gutenberg-mobile/pull/6200 which makes this repo the source of truth for `RNTAztecView.podspec`.

### Testing Instructions for Keyboard
N.A.

## Screenshots or screencast
N.A.
